runtime/compiler/optimizer/EscapeAnalysis.cpp

```diff
@@ -4986,7 +5014,15 @@ bool TR_EscapeAnalysis::fixupNode(TR::Node *node, TR::Node *parent, TR::NodeChec
                TR::Node *andNode = _curTree->getNode()->getFirstChild();
                andNode->removeAllChildren();
 
-               if (TR::Compiler->target.is64Bit())
+               TR_ASSERT(andNode->getOpCodeValue() == TR::iand
```

Comment:
This seems like it should be an assert fatal in the if ladder below processing andNode. Is there anything sensible to be done if we don't have an `iand` or `land` or do we need to bring the VM down?

Reply:
There's probably nothing sensible that we can do.  I'll rearrange the position of the assertion, and make it a `TR_ASSERT_FATAL`